core/tee/tee_svc.c

```diff
@@ -616,8 +629,9 @@ static TEE_Result utee_param_to_param(struct user_ta_ctx *utc,
 
 			p->u[n].mem.mobj = &mobj_virt;
 
-			if (vm_check_access_rights(&utc->uctx, flags, a, b))
-				return TEE_ERROR_ACCESS_DENIED;
+			res = vm_check_access_rights(&utc->uctx, flags, a, b);
+			if (res)
+				goto out;
```

Comment:
For consistency, we need another `goto out` for `p->u[n].mem.size` above.

Reply:
Thanks! updated